.probo.yaml

```diff
@@ -4,5 +4,7 @@ steps:
     makeFile: build-bear.make
     profileName: bear
     runInstall: true
+  - name: Install bear_skin dependencies
+    command: 'cd /var/www/html/sites/all/libraries ; wget https://github.com/Semantic-Org/Semantic-UI/archive/2.1.6.zip; unzip 2.1.6.zip ; mv 2.1.6 semantic_ui ; cp -Rf semantic_ui/dist/* semantic_ui/.'
   - name: Run behat tests
```

Comment:
It's not on the packaging whitelist :( I was trying to add it to get the dependencies working on a probo environment as I was getting that error about Semantic UI library.